src/lib.rs

```diff
@@ -40,6 +40,7 @@ pub trait Config: system::Config + timestamp::Config {
     type Currency: Currency<<Self as system::Config>::AccountId>;
     type MomentsPerDay: Get<Self::Moment>;
     type WeightInfo: WeightInfo;
+    type SilenceDuration: Get<Self::Moment>;
```

Comment:
It is not clear that this is kind of a max value here, I would suggest something like:

* `HeartbeatTimeout`
* `MaxSilenceTime`

Reply:
I prefer MaxSilenceTime